Python/Product/Profiling/Profiling/ProfiledProcess.cs

```diff
@@ -30,11 +30,20 @@ sealed class ProfiledProcess : IDisposable {
         private readonly ProcessorArchitecture _arch;
         private readonly Process _process;
         private readonly PythonToolsService _pyService;
-
+        public string pyArch { get; set; }
```

Comment:
C# properties should be PascalCase. Please capitalize this.

Reply:
Also, if no other classes are using this, it should be private instead of public.

Reply:
@AdamYoblick This value is also set in `Process_OutputDataReceived`, I can't find a way to have `Process_OutputDataReceived` return `e.Data` so I have to set `pyArch` there.

Reply:
Ah that's fine, then don't make it readonly. Just make it a private field. No need to be public because it's not used by a different class, and no need to be a property because you're really not getting any value from the getter nor the setter.

Reply:
ok, done.